rubygem-foreman_azure/rubygem-foreman_azure.spec

```diff
@@ -0,0 +1,79 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_azure
+
+Summary: Azure integration with Foreman (theforeman.org)
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.1
+Release: 1%{?foremandist}%{?dist}
+Group:   Applications/System
+License: GPLv3
+URL:     https://github.com/theforeman/foreman_azure
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: foreman >= 1.9.0
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}rubygems
+Requires: %{?scl_prefix}rubygem(deface) < 2.0
+Requires: %{?scl_prefix}rubygem(fog-azure) = 0.0.2
```

Comment:
I'd suggest loosening this exact restriction in future releases of the foreman_azure gem if you can.